packages/components/src/QueryModel/ViewSelector.tsx

```diff
@@ -0,0 +1,52 @@
+import React, { PureComponent } from 'react';
+import { DropdownButton, MenuItem } from 'react-bootstrap';
+
+import { RequiresModelAndActions } from './withQueryModels';
+import { ViewInfo } from '..';
+
+interface ViewSelectorProps extends RequiresModelAndActions {
+    hideEmptyViewSelector: boolean;
+}
+
+export class ViewSelector extends PureComponent<ViewSelectorProps> {
+    render() {
+        const { model, actions, hideEmptyViewSelector } = this.props;
+        const { isLoading, views, viewName } = model;
+        const activeViewName = viewName !== undefined ? viewName : ViewInfo.DEFAULT_NAME;
```

Comment:
[Nullish coalescing](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-7.html#nullish-coalescing) added with optional chaining, 

const activeViewName = viewName ?? ViewInfo.DEFAULT_NAME;

Reply:
Oh that is awesome. Didn't know about Nullish coalescing. I'll update it.